### ISSUE TITLE
Issue with Queries returning a single parameter (such as `count` queries)

### ISSUE THREAD

**Issue by StephanDecker** [maoosi/prisma-appsync#61]
Great work! We used the beta version and that worked smoothly. After updating to the RC it seems that the graphql queries that return only scalar types don't work anymore (also the generated count* queries do not work). Error message is: "Error reading required parameters from appsyncEvent". After looking at the code, the reason is that an error is thrown because the selectionSetList is empty, see: https://github.com/maoosi/prisma-appsync/blob/main/packages/client/src/adapter.ts#L34-L40

To check (or call one of the count* functions) :
1) Just add a custom resolver to the lambda fn: 
```typescript
            const healthCheck = async () => {
            const result = await app.prismaClient.$queryRaw<{ healthcheck: boolean }[]>`select 1 as healthcheck`;
            return result[0].healthcheck;
        };
```
2) extend your schema:
```graphql
extend type Query {
...
healthCheck: Boolean
 }
```
Additional Info: Event object handed over to lambda:
```
{
  arguments: {},
  identity: null,
  info: {
    fieldName: 'healthCheck',
    selectionSetList: [],
    parentTypeName: 'Query',
    variables: {}
  }
}
```
Thanks for your support.

**Comment by maoosi** [comment]
Thanks for reporting @StephanDecker! I'll release a fix soon.

**Comment by maoosi** [comment]
Fix release in [1.0.0-rc.3](https://github.com/maoosi/prisma-appsync/releases/tag/1.0.0-rc.3).
